File: backend/src/main/java/com/coing/domain/user/controller/dto/UserLoginRequest.java
package com.coing.domain.user.controller.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;

public record UserLoginRequest(
	@NotBlank(message = "{email.required}")

	@Email(message = "{invalid.email.format}") String email,

	@NotBlank(message = "{password.required}") String password
) {
}

File: backend/src/main/java/com/coing/domain/user/controller/dto/UserLoginRequest.kt
package com.coing.domain.user.controller.dto

import com.coing.global.annotation.NoArg
import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

@NoArg
data class UserLoginRequest(
    @field:NotBlank(message = "{email.required}")
    @field:Email(message = "{invalid.email.format}")
    val email: String,

    @field:NotBlank(message = "{password.required}")
    val password: String
)
